utils/zerovec/src/varzerovec/ule.rs

```diff
@@ -153,11 +136,17 @@ where
     }
 }
 
-impl<T: AsVarULE> fmt::Debug for VarZeroVecULE<T>
+impl<T: VarULE + ?Sized> fmt::Debug for VarZeroVecULE<T>
 where
-    T::VarULE: fmt::Debug,
+    T: fmt::Debug,
 {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_list().entries(self.iter()).finish()
     }
 }
+
+impl<T: ?Sized> AsRef<VarZeroVecULE<T>> for VarZeroVecULE<T> {
```

Comment:
Question: Why do you need this impl?  Isn't it always the case that `X` implements `AsRef<X>` ?

Reply:
Nope!